include/rwqueue.h

```diff
@@ -99,6 +99,10 @@ class RWQueue {
 	//  return false and the item will not be queued.
 	bool Enqueue(T&& item);
 
+	// Returns false and does nothing if the queue is at capacity or the queue is not running
+	// Otherwise the item gets moved into the queue and it returns true
+	bool NonblockingEnqueue(T&& item);
```

Comment:
Copied or moved?

Reply:
Moved, same as the existing `Enqueue` method.